[ADD] Timer Finished

Pedroca2005BR opened the pull request:
Finished the Stopwatch and added some public functions to make its use as easy as possible.

Pedroca2005BR committed:
[ADD] Timer Finished

Pedroca2005BR committed:
[FIX] ResetTimer

Pedroca2005BR commented:
https://www.canva.com/design/DAGGP_ZIrBk/cNYVp-RTDQ2nkEy5hYVOig/edit?utm_content=DAGGP_ZIrBk&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

KamalFR commented on Assets/Stopwatch/HelperScript.cs:
```diff
@@ -0,0 +1,11 @@
+{
+    public void AccerelateTime()
+    {
+        Time.timeScale += 10;
```
- Possivelmente colocar um limite na aceleração (o editor da Unity crasha depois que o valor fica maior que 100)

KamalFR reviewed (commented):
- Mudar o texto do canva (onde ta documentação) para inglês

Pedroca2005BR committed:
[DEL] Helper script

The helper script was there just to help testing the application. It isn't needed for the final product.

Pedroca2005BR commented:
Canva alterado para linguagem inglesa
Helper script apagado (explicação no commit)
